Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM eclipse-temurin:17-jdk-jammy
 
-RUN apt-get update && apt-get -y install python3
+RUN apt-get update && apt-get -y install python3 && apt-get -y install nodejs
```

Comment:
How about installing node from NodeSource? We can specify node version that way.
https://github.com/nodesource/distributions